### ISSUE TITLE
Global styling for video player like Plyr.io

### ISSUE THREAD

**Issue by SeeJaneB** [soflyy/oxygen-bugs-and-features#1510]
It would be nice to have control over the appearance and behavior of videos on an Oxygen site that would apply to locally hosted videos, along with external videos from YouTube or Vimeo. I'm thinking custom colors, and the ability to take advantage of certain parameters that are available to reduce the clutter that you get from external embeds, for example related videos or more videos and excessive brand marks. 

There are ways to do some of these things manually by adding parameters to an embed code, but it would be better to have a way to control this in the global settings so you can set up your video player to match the style for the particular website and not have to edit every embed code. Additionally, in my case, I need to use the dynamic data option for the Oxygen video element to create a template that will just pull the YouTube link from the custom field that the client will enter. 

I believe this would go along with one of the main draws of Oxygen, in that you have complete control of every detail of your design and having global settings can help with uniformity across the site and a more efficient workflow. I also think it would provide the ability to make websites look that much more professional with this added attention to detail.

I believe there is possibly a way already integrate this functionality provided by [Plyr.io](Plyr.io) and their in depth details at github [https://github.com/sampotts/plyr](https://github.com/sampotts/plyr). I really like the look and sounds of what they are offering but have no idea where to begin with implementation, nor do I know if it would apply to videos that are added via the Oxygen video Element. I would love for Oxygen to just build this in for many users like myself that want control over the details and are learning some code, but not experts yet.

It seems like most plugins that claim to do this type of thing operate with generating shortcodes and posting them where you want on the site, which doesn't work for easy entry on the client end. I experimented with Embed Plus [https://wordpress.org/plugins/youtube-embed-plus/](https://wordpress.org/plugins/youtube-embed-plus/) and the ARVE [https://wordpress.org/plugins/advanced-responsive-video-embedder/](https://wordpress.org/plugins/advanced-responsive-video-embedder/) plugin, since they operate with just entering the link into a WYSWYG editor. This seemed to work to enter the link in a custom field, and I was able to pull the data into the template on the back end using a text field, but it didn't display on the front end, nor did it apply globally to videos that were added via the video element.

The bottom line is that I really just want the video player in Oxygen to be like the demo on Plyr.io with all the CSS customizations they provide in the table in their documentation on github built right into Oxygen under the global settings, or at least available under the video element so a class can be created for uniformity across an entire website.
